flyway/src/main/scala/com/avast/sst/flyway/FlywayConfig.scala

```diff
@@ -0,0 +1,25 @@
+package com.avast.sst.flyway
+
+import java.nio.charset.{Charset, StandardCharsets}
+
+import org.flywaydb.core.api.MigrationVersion
+
+final case class FlywayConfig(baselineOnMigrate: Boolean = false,
+                              baselineVersion: Option[MigrationVersion] = None,
```

Comment:
shouldn't the default be `EMPTY`? (I have no idea, don't really understand the issue at hand, just asking questions :) )

Reply:
Actually the default is "1" but I don't want to have that in the config so I decided to make it optional and not set it in this case.